packages/filecmd.rb

```diff
@@ -1,74 +1,83 @@
 require 'package'
 
 class Filecmd < Package
-  description 'file command determines the file type.'
+  description 'file and libmagic determine file type'
   homepage 'http://ftp.astron.com/'
-  @_ver = '5.40'
-  version "#{@_ver}-2"
+  @_ver = '5.43'
+  version @_ver
   license 'BSD-2 and GPL-3+' # Chromebrew's filefix is GPL-3+, file itself is BSD-2
   compatibility 'all'
   source_url "http://ftp.astron.com/pub/file/file-#{@_ver}.tar.gz"
-  source_sha256 '167321f43c148a553f68a0ea7f579821ef3b11c27b8cbe158e4df897e4a5dd57'
-
-  binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/filecmd/5.40-2_armv7l/filecmd-5.40-2-chromeos-armv7l.tpxz',
-    armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/filecmd/5.40-2_armv7l/filecmd-5.40-2-chromeos-armv7l.tpxz',
-    i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/filecmd/5.40-2_i686/filecmd-5.40-2-chromeos-i686.tpxz',
-    x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/filecmd/5.40-2_x86_64/filecmd-5.40-2-chromeos-x86_64.tpxz'
-  })
-  binary_sha256({
-    aarch64: '651c9ebb6b6294969df3f776b49fdb3eb08371a7ab8eb929a8e2c8d298e9d6bd',
-    armv7l: '651c9ebb6b6294969df3f776b49fdb3eb08371a7ab8eb929a8e2c8d298e9d6bd',
-    i686: 'ea5a96dbdf900f836b1ca69eefa72b2622722a1f3d0e46736932cc75b12a653f',
-    x86_64: '4d0da2febbb0f5a9e5d08dfe2026a9816aa2b1fb603d2c9328082e740a96c8a2'
-  })
+  source_sha256 '8c8015e91ae0e8d0321d94c78239892ef9dbc70c4ade0008c0e95894abfb1991'
 
   depends_on 'bz2' # R
   depends_on 'glibc' # R
+  depends_on 'libseccomp' # R
   depends_on 'xzutils' # R
   depends_on 'zlibpkg' # R
 
-  def self.build
-    # The filefix command changes the full path of the file command in configure scripts.
-    # Execute this command from your source code root directory.
+  def self.prebuild
+    # The filefix command changes the full path of the file command in configure scripts
+    # Execute this command from your source code root directory
+    # This may be needed in older autotools tarballs due to an old libtool bug
+    # It's better to run filefix if unsure
+    # See https://savannah.gnu.org/support/?func=detailitem&item_id=110550 for more information
+
     @filefix = <<~EOF
-      #!/bin/bash
-      for f in $(find . -name configure); do
-        sed -i 's,/usr/bin/file,#{CREW_PREFIX}/bin/file,g' ${f}
-      done
+      #!/usr/bin/env bash
+      while IFS= read -r -d '' f; do
+        sed -i 's,/usr/bin/file,#{CREW_PREFIX}/bin/file,g' "${f}"
+      done <  <(find . -name configure -print0)
```

Comment:
No need to rewrite but whatever. :)

Reply:
https://www.shellcheck.net/wiki/SC2044

Reply:
If we ever use spaces or brackets in filenames then we have a bigger problem but this is good to know.